tests/specs/spa/harvesting.e2e.js

```diff
@@ -2,7 +2,7 @@ import { checkSpa } from '../../util/basic-checks'
 import { testInteractionEventsRequest } from '../../../tools/testing-server/utils/expect-tests'
 import { JSONPath } from 'jsonpath-plus'
 
-describe('spa harvesting', () => {
+describe.withoutCoverage()('spa harvesting', () => {
```

Comment:
example of skipping an entire describe block.  Could also be applied to individual `it(...)` calls